### ISSUE TITLE
Help setup Lake build/clean tasks

### ISSUE THREAD

**Issue by lovettchris** [leanprover/vscode-lean4#61]
Help users configure .vscode/tasks.json so it can do things like this, so the user can use Ctrl+Shift+B to build, and something to clean, and perhaps even run a compiled app, depending on the type of lake project, etc...

```json
{
    // See https://go.microsoft.com/fwlink/?LinkId=733558
    // for the documentation about the tasks.json format
    "version": "2.0.0",
    "tasks": [
        {
            "label": "build",
            "type": "shell",
            "command": "lake build"
        },
        {
            "label": "clean",
            "type": "shell",
            "command": "lake clean"
        }
    ]
}
```

**Comment by leodemoura** [comment]
This sounds like a useful feature to have, but I am an Emacs user :) 
@gebner @kha @Vtec234 @EdAyers @tydeu Any thoughts?

**Comment by lovettchris** [comment]
Here's more info: https://code.visualstudio.com/docs/editor/tasks

**Comment by mhuisi** [comment]
This was implemented by #334, though using commands, not tasks. Tasks cannot be parameterized and you cannot obtain stdout/stderr output, making them too inconvenient.
